src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -77,10 +80,37 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
           ) : (
             <>
               <FontAwesomeIcon icon={faChevronLeft} size="sm" />
-              <FontAwesomeIcon icon={faCommentDots} size="lg" />
+              <div className="chat-sidebar__controller-wrapper">
+                <FontAwesomeIcon icon={faCommentDots} size="lg" />
+                <span className="chat-sidebar__controller-text">Chat</span>
+              </div>
             </>
           )}
         </button>
+        {!isExpanded && (
+          <button
+            aria-label={isExpanded ? "Hide chat" : "Show chat"}
+            className="chat-sidebar__controller chat-sidebar__private-chat"
+            onClick={togglePrivateChatSidebar}
+          >
+            {isExpanded ? (
+              <FontAwesomeIcon icon={faChevronRight} size="sm" />
+            ) : (
+              <>
+                <FontAwesomeIcon icon={faChevronLeft} size="sm" />
+                <div className="chat-sidebar__controller-wrapper">
+                  <FontAwesomeIcon icon={faEnvelope} size="lg" />
+                  <span className="chat-sidebar__controller-text">
+                    Messages
```

Comment:
Comparing burn environment and local environment with your changes.

1. Text and icons at the burn look smaller thus have more free space.
2. There are the same icons in the chat tabs which help us distinguish the buttons (see the screenshot). - should we add them? or as we made the chat thinner, we can't?
3. In the closed state, the button at the burn environment is wider. 
 
BURN
![image](https://user-images.githubusercontent.com/63313732/135054551-6f2dc5ac-bc6f-40bb-b263-765ef6e82909.png)

LOCAL
![image](https://user-images.githubusercontent.com/63313732/135055118-94fb1fe0-74e7-4f2a-bf9b-78b0b2620eb8.png)

Reply:
also, if you make the text smaller, we'll have enough space for the red circle.
![image](https://user-images.githubusercontent.com/63313732/135056244-f871c549-4439-47ee-8dae-3ca873e4ee87.png)

Reply:
We need the red dot over the envelop icon not over the text please .
@alexey-petrov

Reply:
@ermiaSparkle moved that dot a little higher 
![image](https://user-images.githubusercontent.com/56254806/135814033-f63c36a6-65ab-4c6a-9bb0-04f123bb154d.png)
@sunny-viktoryia made text smaller, opened state arrow width is now a little wider
since we've changed chat opened state width to 300px adding icons there might not be necessary as they would take too much space, but I guess depends on what you guys say 
fb21560

Reply:
@alexey-petrov can you show this circle according to the requirements here?
https://github.com/sparkletown/internal-sparkle-issues/issues/1271

(and, please, link this issue in the description of the PR)

Reply:
@sunny-viktoryia fixed in f72eceb